packages/client/src/comp/states/Base.test.tsx

```diff
@@ -38,11 +43,19 @@ const fillWithoutSigning = ({getByLabelText, getByTestId}: RenderResult) => {
         value: '123-456-7890'
       },
     })
+  })
+}
 
+/** Triggers validation that should happen after form is filled */
```

Comment:
Can we test if an unregistered voter?

- mock alloy call to return unregistered
- check that modal popsup via jest data-testid